### PR TITLE
Add Requirement template

### PR DESCRIPTION
**Pull request by govindsi**

### CODE REVIEW

**Review by hrishiballal — CHANGES_REQUESTED** [review]

File: requirements/usv_req.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Requirements
+
+## Table of Regulatory Security Requirements for UTM services
+
+| Regulatory Requirement                      | Standard        | Asset                                                                                      | State of the Art           | Verifier Requirement ID | Test| Report |
```
[the remark this answers is not in the document]
Comment (govindsi) [reply requirements/usv_req.md]: done

[the remark this answers is not in the document]
Comment (govindsi) [reply requirements/usv_req.md]: done

[the remark this answers is not in the document]
Comment (govindsi) [reply requirements/usv_req.md]: done

File: requirements/usv_req.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Requirements
+
+## Table of Regulatory Security Requirements for UTM services
+
+| Regulatory Requirement                      | Standard        | Asset                                                                                      | State of the Art           | Verifier Requirement ID | Test| Report |
+|---------------------------------------------|-----------------|--------------------------------------------------------------------------------------------|----------------------------|-----------------|------------|--------|
+| Article 4 - Registration and Identification | NA              | UAS Operator Registration Data, Operator Identity, UAS Identification Information          | TLS 1.3, Oauth2.0, Signing | VRID-001        | VTID-001   |        |
+| Article 8 - Network Identification          | ASTM F3411-22a  | Network Identification Data                                                                | TLS 1.3, Oauth2.0, Signing | VRID-002        | VTID-002   |        |
```
[the remark this answers is not in the document]
Comment (govindsi) [reply requirements/usv_req.md]: i was referring to state of art mitigation, i have mapped with CIA. I am thinning details we will outline separately.

File: requirements/usv_req.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+|---------------------------------------------|-----------------|--------------------------------------------------------------------------------------------|----------------------------|-----------------|------------|--------|
+| Article 4 - Registration and Identification | NA              | UAS Operator Registration Data, Operator Identity, UAS Identification Information          | TLS 1.3, Oauth2.0, Signing | VRID-001        | VTID-001   |        |
+| Article 8 - Network Identification          | ASTM F3411-22a  | Network Identification Data                                                                | TLS 1.3, Oauth2.0, Signing | VRID-002        | VTID-002   |        |
+| Article 9 - Geo-awareness                   | ED-318          | Geo-awareness Data                                                                         | TLS 1.3, Oauth2.0, Signing | VRID-003        | VTID-003   |        |
+| Article 10 - UAS Flight Authorization       | ASTM F3548-21   | Flight Authorization Data                                                                  | TLS 1.3, Oauth2.0, Signing | VRID-004        | VTID-004   |        |
```
[the remark this answers is not in the document]
Comment (govindsi) [reply requirements/usv_req.md]: done

File: requirements/usv_req.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+| Article 4 - Registration and Identification | NA              | UAS Operator Registration Data, Operator Identity, UAS Identification Information          | TLS 1.3, Oauth2.0, Signing | VRID-001        | VTID-001   |        |
+| Article 8 - Network Identification          | ASTM F3411-22a  | Network Identification Data                                                                | TLS 1.3, Oauth2.0, Signing | VRID-002        | VTID-002   |        |
+| Article 9 - Geo-awareness                   | ED-318          | Geo-awareness Data                                                                         | TLS 1.3, Oauth2.0, Signing | VRID-003        | VTID-003   |        |
+| Article 10 - UAS Flight Authorization       | ASTM F3548-21   | Flight Authorization Data                                                                  | TLS 1.3, Oauth2.0, Signing | VRID-004        | VTID-004   |        |
+| Article 11 - Traffic Information            | NA              | Traffic Information Data                                                                   | TLS 1.3, Oauth2.0, Signing | VRID-005        | VTID-005   |        |
```
[the remark this answers is not in the document]
Comment (govindsi) [reply requirements/usv_req.md]: i was referring to state of art mitigation, i have mapped with CIA. I am thinning details we will outline separately.

File: requirements/usv_req.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+| Article 13 - Conformance Monitoring         | ASTM F3548-21   | Conformance Monitoring Data                                                                | TLS 1.3, Oauth2.0, Signing | VRID-006        | VTID-006   |        |
```
[the remark this answers is not in the document]
Comment (govindsi) [reply requirements/usv_req.md]: i have mapped to NA

**Review by hrishiballal — APPROVED** [review]

**Pull request merged**